keyboards/coseyfannitutti/mullet/mullet.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2019 COSEYFANNITUTTI
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+#define _x_ KC_NO
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT( \
```

Comment:
THis looks like it matches the 65_ansi layout.  If so, you may want to add support for that:

https://github.com/qmk/qmk_firmware/blob/master/layouts/default/65_ansi/default_65_ansi/keymap.c

Reply:
It's a little bit different in that it does not have the right control. There is a gap on mine and that key isn't possible. Mine is 67 key.

Reply:
Also updated the readme for mullet to include image.